dce/main.go

```diff
@@ -118,8 +119,10 @@ func (exec *dockerComposeExecutor) LaunchTask(driver exec.ExecutorDriver, taskIn
 	// Wait for pod launching until timeout
 	var ctx context.Context
 	var cancel context.CancelFunc
-	ctx = context.Background()
-	ctx, cancel = context.WithTimeout(ctx, config.GetLaunchTimeout()*time.Millisecond)
+
+	ctx, cancel = context.WithTimeout(context.Background(), config.GetLaunchTimeout()*time.Millisecond)
+	ctx = context.WithValue(ctx, types.APP_START_TIME, appStartTime)
```

Comment:
Where does context value ```APP_START_TIME``` be used?

Reply:
It was getting used in the earlier design. Will remove it. thanks.